src/api/AzureResourceGroupsExtensionApi.ts

```diff
@@ -41,19 +41,19 @@ export class InternalAzureResourceGroupsExtensionApi implements AzureHostExtensi
         return this.#appResourceTree;
     }
 
-    public get appResourceTreeView(): TreeView<AzExtTreeItem> {
+    public get appResourceTreeView(): TreeView<unknown> {
```

Comment:
Won't users of `0.0.1` still expect `AzExtTreeItem`-derived views?  Or, is the compat shim replacing them with something that just looks like an `AzExTreeItem` but isn't?